tron/config/schema.py

```diff
@@ -226,7 +226,7 @@ def config_object_factory(name, required=None, optional=None):
     'StatePersistenceTypes', dict(shelve='shelve', yaml='yaml', dynamodb='dynamodb')
 )
 
-ExecutorTypes = Enum('ExecutorTypes', dict(ssh='ssh', mesos='mesos'))
+ExecutorTypes = Enum('ExecutorTypes', dict(ssh='ssh', mesos='mesos', spark='spark'))
```

Comment:
My personal opinion is that tron shouldn't know about a special spark thing. I was hoping it could launch it directly on mesos, so it could just "be" a mesos executor.